tests/secrets/test_secrets_verification.py

```diff
@@ -45,6 +44,7 @@ def test_verify_secrets_insufficient_params_no_flag() -> None:
 
 
 @responses.activate
+@mock.patch.dict(os.environ, {"CKV_VALIDATE_SECRETS": "true"})
```

Comment:
why remove here the `BC_API_KEY` ? :)

Reply:
It is an unnecessary key, the condition to run secrets verify is on `bc_integration.bc_api_key` and not on the environment variable